plugins/mcreator-core/features/vertical_anchor_below_top.json

```diff
@@ -0,0 +1,18 @@
+{
+  "args0": [
+    {
+      "type": "field_number",

```

Comment:
Do negative offsets work in those parameters?

Reply:
They do, tested with the simple block feature and positions outside the world bound are simply ignored